src/olympia/lib/settings_base.py

```diff
@@ -1119,6 +1119,11 @@ def get_db_config(environ_var, atomic_requests=True):
     'olympia.scanners.tasks.run_customs': {'queue': 'devhub'},
     'olympia.scanners.tasks.run_wat': {'queue': 'devhub'},
     'olympia.scanners.tasks.run_yara': {'queue': 'devhub'},
+    'olympia.scanners.mark_yara_query_rule_as_completed': {'queue': 'devhub'},
+    'olympia.scanners.run_yara_query_rule': {'queue': 'devhub'},
+    'olympia.scanners.run_yara_query_rule_on_versions_chunk': {
+        'queue': 'devhub'
+    },
```

Comment:
Shouldn't we have a new queue for the code search (so that the regular devhub tasks are not impacted)?

Reply:
Yeah, probably. I'll file a follow-up to change that, needs a discussion with ops to see what workers we prioritize (IIRC most queues outside the "priority" one go the same pool of workers anyway)

Reply:
https://github.com/mozilla/addons-server/issues/13267